### DISCUSSION TITLE
I have delete some element by CAD, but ezdxf still can read them, why?

### DISCUSSION THREAD

**Discussion by xinkun-mini** [mozman/ezdxf#1204] [Q&A]
here is a arrow at (7400,32784)  with no elements else below
![image](https://github.com/user-attachments/assets/7454b5cb-20a1-4fea-8a96-b76fd55c8ca2)

but using ezdxf api, I can still get a insert object at position (5000,5000) , but I have already remove it from CAD , and I can't see it too

![image](https://github.com/user-attachments/assets/2276b6ed-f7a5-453f-b7f5-6bd530e3f350)

so I want to know if a insert object is 'deleted', I wonder to know if there will be a state field 

thanks for all!!

**Comment by mozman** [comment]
That's not enough information to give an answer, but you can check if an entity is alive by looking at the `entity.is_alive` property. From your screenshot I see that the `dxf` namespace is not `None`, so this `INSERT` entity is still alive. If you want to know why this entity still exists you'll have to ask your CAD vendor, `ezdxf` only shows what is stored in the DXF file.
